lib/Validator.js

```diff
@@ -312,7 +310,7 @@ Validator.prototype._validateNewlineMaximum = function() {
 							.replace('{b}', self._settings.newlineMaximum);
 
 						data = {message: message};
-						data = merge(MESSAGES.NEWLINE_MAXIMUM, data);
+						data = merge({}, MESSAGES.NEWLINE_MAXIMUM, data);
```

Comment:
I noticed this while debugging the tests. Without the leading empty object, it actually overrides the NEWLINE_MAXIMUM string with hardcoded values.